tools/wutool/wutool.cpp

```diff
@@ -40,37 +40,142 @@
 
 static unsigned testSize = 1000;
 
-void usage()
+void usage(const char * action = nullptr)
 {
-    printf("Usage: wutool action [WUID=xxx] [DALISERVER=ip] [CASSANDRASERVER=ip] [option=value]...\n\n"
-           "Actions supported are:\n"
-           "   list <workunits>    - List workunits\n"
-           "   dump <workunits>    - Dump xml for specified workunits\n"
-           "   delete <workunits>  - Delete workunits\n"
-           "   results <workunits> - Dump results from specified workunits\n"
-           "\n"
-           "   archive <workunits> - Archive to xml files [TO=<directory>] [DEL=1] [DELETERESULTS=1] [INCLUDEFILES=1]\n"
-           "   restore <filenames> - Restore from xml files [INCLUDEFILES=1]\n"
-            "\n"
-           "   orphans             - Delete orphaned information from store\n"
-           "   cleanup [days=NN]   - Delete workunits older than NN days\n"
-           "   validate [fix=1]    - Check contents of workunit repository for errors\n"
-           "   clear               - Delete entire workunit repository (requires entire=1 repository=1)\n"
-           "   initialize          - Initialize new workunit repository\n"
-           "\n"
-           "If CASSANDRASERVER is specified, you can specify some connection options including:\n"
-           "   CASSANDRA_KEYSPACE   - default is hpcc\n"
-           "   CASSANDRA_USER\n"
-           "   CASSANDRA_PASSWORD\n"
-           "   CASSANDRA_PARTITIONS - used if creating a new repository\n"
-           "   CASSANDRA_PREFIXSIZE - used if creating a new repository\n"
-           "   TRACELEVEL\n"
-           "<workunits> can be specified on commandline, or can be specified using a filter owner=XXXX. If omitted,\n"
-           "all workunits will be selected.\n"
-            );
+    if (action && strieq(action, "info"))
+    {
+        printf("Usage: wutool info <workunits> <filter>\n"
```

Comment:
@ghalliday As multiple filters are supported, shouldn't it be: 
`Usage: wutool info  <workunits> <filter>[,<filter>...]`

Reply:
I have reworded the next line instead.  Does the new working capture the information?